invenio/modules/workflows/static/css/workflows/workflows.css

```diff
@@ -113,6 +113,16 @@ table tr:hover a.faded {
     font-family: FontAwesome;
 }
 
+.active-editable{
```

Comment:
Let's add this to the `inspire-next` overlay instead of Invenio. Ping me, and I can show you how we can do it.